web/packages/teleport/src/Discover/Database/DeployService/AutoDeploy/SelectSecurityGroups.tsx

```diff
@@ -105,11 +113,29 @@ export const SelectSecurityGroups = ({
 
   return (
     <>
+      <Flex alignItems="center" gap={1} mb={2}>
+        <Subtitle3>Select Security Groups</Subtitle3>
+        <ToolTipInfo>
+          <Text>
+            The security groups you pick must meet the following requirements:
+            <ul>
+              <li>
+                all security groups must allow outbound connectivity (eg:
+                0.0.0.0/0) to reach Teleport cluster
+              </li>
+              <li>
+                all security groups must allow inbound connectivity from the ECS
+                task security group or from the ECS task subnet(s) CIDR blocks
+              </li>
+            </ul>
```

Comment:
```suggestion
            Select security group(s) based on the following requirements:
            <ul>
              <li>
                The selected security group(s) must allow all outbound traffic
                (eg: 0.0.0.0/0)
              </li>
              <li>
                A security group attached to your database(s) must allow inbound
                traffic from a security group you select or from all IPs in the
                subnets you selected
              </li>
```

Security groups are additive, so only one of the ECS task security groups needs to allow (0.0.0.0/0) outbound.
No inbound rules are needed for these either, but the databases' security groups need to allow inbound from the ECS task.